modules/helm/crds.mk

```diff
@@ -50,9 +50,8 @@ generate-crds: | $(NEEDS_CONTROLLER-GEN) $(NEEDS_YQ)
 	# TODO(@SgtCoDFish): This is a temporary fix for an accidentally created nested crds directory
 	# in the PR: https://github.com/cert-manager/makefile-modules/pull/239
 	# Once this has been run on every repo with the mistaken folder, this can be removed
-	rm -r $(crds_dir)/crds
+	rm -rf $(crds_dir)/crds
 
-	rm -rf $(crds_gen_temp)
```

Comment:
This line was correct, I think, shouldn't we remove the line above and the comment?

Reply:
This is why we have code reviews! thank you!

Reply:
Fixed!